tests/expectations/core/conftest.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import pandas as pd
+import pytest
+
+from great_expectations.data_context.util import file_relative_path
+
+
+@pytest.fixture(scope="module")
+def titanic_df() -> pd.DataFrame:
```

Comment:
Do we currently use any type of immutable dataframe? That would enforce that this object won't be modified when shared between tests.

Reply:
Hi @billdirks I dont think we use any immutable dataframes currently, and I really like the idea. Does pandas have a way of specifying this internally? I did a google search and found something called [static frame](https://static-frame.readthedocs.io/en/latest/articles/upgrade.html), which claims to do what you are describing.